File: Arreglos/Java/Columnas_D_S.java
package Java;
public class Columnas_D_S {
    
    public static void main (String[] args){
        int dimension = 5;
        int arreglo[][]= new int [dimension][dimension];
        int filas=0, columnas=0;
    
        for (columnas=0;columnas<dimension; columnas ++){
            for (filas=0; filas<dimension; filas++){
                if (filas<columnas){
                    System.out.print("      ");
                }else{
                    System.out.print("("+columnas+","+filas+") ");
                }
                
             }   
        System.out.println(" "); 
        }
    }
    
}

File: Arreglos/C/Columnas_D_S.c
/*
Author: Velentina Ortiz Porras
Date: 15/02/2022
Description: Leer un arreglo donde imprima los valores hacia arriba de la diagonal principal por columnas 
*/

#include <stdio.h>

int main(int argc, char const *argv[])
{
int dimension = 5;
 int filas, columnas;

 for (filas = 0; filas < dimension; filas++){

     for (columnas = 0; columnas <= filas; columnas++){

        if(filas + columnas <= 8){
         printf("[%d,%d]", filas, columnas);
        }
        else {
          printf("     "); 
        }     
     }
     printf("\n");
 }
    
    return 0;
}
